pkg/querier/queryrange/roundtrip.go

```diff
@@ -68,6 +71,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&cfg.AlignQueriesWithStep, "querier.align-querier-with-step", false, "Mutate incoming queries to align their start and end with their step.")
 	f.BoolVar(&cfg.CacheResults, "querier.cache-results", false, "Cache query results.")
 	f.BoolVar(&cfg.ShardedQueries, "querier.parallelise-shardable-queries", false, "Perform query parallelisations based on storage sharding configuration and query ASTs. This feature is supported only by the chunks storage engine.")
+	f.Var(&cfg.ForwardHeaders, "querier.forward-headers-list", "List of headers forwarded by the query Frontend to downstream querier.")
```

Comment:
I understand the CLI flag prefix (`-querier.`) has been copied from others, but this flag practically apply to the query-frontend so I would change the prefix to `-frontend.`

Reply:
Make sense.